### PR TITLE
fix(sec): upgrade org.fusesource.mqtt-client:mqtt-client to 1.15

### PR DESCRIPTION
**Pull request by zhoumengyks**
### What happened？
There are 1 security vulnerabilities found in org.fusesource.mqtt-client:mqtt-client 1.10
- [CVE-2019-0222](https://www.oscs1024.com/hd/CVE-2019-0222)

### What did I do？
Upgrade org.fusesource.mqtt-client:mqtt-client from 1.10 to 1.15 for vulnerability fix

### What did you expect to happen？
Ideally, no insecure libs should be used.

### The specification of the pull request
[PR Specification](https://www.oscs1024.com/docs/pr-specification/) from OSCS

### CODE REVIEW

File: examples/storm-mqtt-examples/pom.xml [hunk 1]
```diff
@@ -13,9 +14,7 @@
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
--->
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (bipinprasad) [inline examples/storm-mqtt-examples/pom.xml:17]: Can you insert a new line after the comment end to be consistent with the other pom files.

File: external/storm-mqtt/pom.xml [hunk 2]
```diff
@@ -13,8 +14,7 @@
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (bipinprasad) [inline external/storm-mqtt/pom.xml:17]: Here as well: a new line after the end of the copyright comment - for consistency.

File: external/storm-mqtt/pom.xml [hunk 3]
```diff
@@ -73,7 +73,7 @@
         <dependency>
             <groupId>org.fusesource.mqtt-client</groupId>
             <artifactId>mqtt-client</artifactId>
-            <version>1.10</version>
+            <version>1.15</version>
```
Comment (bipinprasad) [inline external/storm-mqtt/pom.xml:76]: Also Update the DEPENDENCY and LICENSE files so that the build will go to success.

**Comment by rzo1** [comment]
Included in https://github.com/apache/storm/pull/3463

**Pull request closed without merging**